components/raftstore-v2/src/operation/command/admin/compact_log.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Let's unify the file name and related struct/methods to raft log gc.

Reply:
What do you mean? compact_log -> raft_log_gc?

Reply:
Yes.